### ISSUE TITLE
Make Drawers and Tabs persistent when selecting blocks

### ISSUE THREAD

**Issue by razwan** [pixelgrade/nova-blocks#220]
On multiple occasions, I discussed with @georgeolaru and came to the conclusion that an important UX improvement would be to keep a settings drawer open, and the same tab active when navigating between blocks of the same block type. This can also be done for blocks of different blockType but which share the same Controls Sections.

**Comment by georgeolaru** [comment]
@razwan I tested this feature, and it's working great! 

Although it feels natural as it is, I stumbled upon the feeling that I don't know which other blocks have support for the current section. I identified two things that I want to do (input) and explored a few options for them (output):

![image](https://user-images.githubusercontent.com/1632775/95725023-c7f47a00-0c7f-11eb-8caf-7484ca16c7d8.png)

---

![image](https://user-images.githubusercontent.com/1632775/95725451-5832bf00-0c80-11eb-8e1c-78b49fc42444.png)


---

I don't think a simple signifier (see below) would be of much help here; rather, we might need to take a broader perspective on how these sections are used and linked from blocks, to pages, and to the whole site.

![Screenshot2020-10-12 at 10 03](https://user-images.githubusercontent.com/1632775/95715524-afca2e00-0c72-11eb-9e72-6ab3aa2c0e3c.gif)

---

@razwan I suggest sticking with your current implementation as it works as we intended, get rid of any bugs that you found while testing, and close this issue. Nice and quick work 👏

**Comment by razwan** [comment]
Hey @georgeolaru 

The linking of the style of two or more blocks is kinda tricky. We could do it the way we did when we explored the Block Style Manager plugin. But it seems to me that we would need some sort of _layers_ of styles being registered for each section (this means that settings would be exclusive for each section and there would be no blurred lines). 

I'm not really fond of the signifier. I personally don't feel the need to know if a block also has a specific set of attributes I'm working with at any moment, even though I understand how this could be confusing. Also, the small dot doesn't give any useful info and it looks rather like an _altered_ state. It would seam more straight-forward for me if every settings section would have it's own _icon_ and we would display all of them instead of the simple dot, each with various states like active, unavailable or altered.

![Frame 1](https://user-images.githubusercontent.com/1633371/95742763-2ed25d00-0c99-11eb-9c87-99dfd046dcf0.png)

Regarding this issue we're, I'll merge the pull request I made and close this for the time being.
